teamB/eungSooKim/app.js

```diff
@@ -29,22 +31,16 @@ database
 app.use(express.json());
 app.use(cors());
 app.use(morgan("tiny"));
-
-
-app.use(express.json());
-app.use(cors());
-app.use(morgan('tiny'));
-
-const server = http.createServer(app)
+const server = http.createServer(app);
 const PORT = process.env.PORT;
 
 app.get("/ping", (req, res) => {
   res.json({ message: "pong" });
 });
 
 
-app.post("/users", async (req, res, next) => {
-  const { name, email, password } = req.body;
+app.post("/users/signup", async (req, res) => {
+  const { name, email, passWord } = req.body;
```

Comment:
```suggestion
  const { name, email, password } = req.body;
```

다소 마이너한 리뷰이긴하지만 password 자체가 pass + word의 합성어가 아닌 하나의 단어이기 때문에 CameCase를 적용하지 않아도 될 것 같네요 :)